templates/details/overview.html

```diff
@@ -13,6 +13,29 @@ <h4 class="p-heading--5 u-no-margin--bottom">About</h4>
         </p>
         <hr class="p-separator--shallow">
       {% endif %}
+      {% if package.type == "bundle" %}
+        {% if not package.result.summary %}
+          <h4 class="p-heading--5 u-sv1 u-no-margin--bottom">About</h4>
+        {% endif %}
+          <div class="p-topology-info" data-js="topology-info">
+            <div class="p-card--highlighted">
+              <object data="https://api.jujucharms.com/charmstore/v5/bundle/{{ package.name }}/diagram.svg" type="image/svg+xml" style="pointer-events: none;"></object>
+              <i class="p-icon--fullscreen" role="button" tabIndex="0">Show fullscreen</i>
```

Comment:
This is not keyboard accessible can you make this a base button and attach the open click event listener to it please?